src/Symfony/Component/DependencyInjection/Tests/Fixtures/containers/container9.php

```diff
@@ -64,7 +64,6 @@
 ;
 $container
     ->register('baz', 'Baz')
-    ->addMethodCall('setFoo', array(new Reference('foo_with_inline')))
```

Comment:
Readding the method call leads to a circular reference detected by the added pass (`foo_with_inline` -> `inlined` -> `baz` -> `foo_with_inline`).